Makefile

```diff
@@ -10,9 +10,15 @@ DOCKER_CACHE       := $(CURDIR)/.cache
 ANTCTL_BINARY_NAME ?= antctl
 OVS_VERSION        := $(shell head -n 1 build/images/deps/ovs-version)
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
+CNI_BINARIES_VERSION := $(shell head -n 1 build/images/deps/cni-binaries-version)
+NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
+WIN_BUILD_TAG      := $(shell md5sum build/images/base-windows/image-tag| head -c 32)
 
 DOCKER_BUILD_ARGS = --build-arg OVS_VERSION=$(OVS_VERSION)
 DOCKER_BUILD_ARGS += --build-arg GO_VERSION=$(GO_VERSION)
+DOCKER_BUILD_ARGS += --build-arg CNI_BINARIES_VERSION=$(CNI_BINARIES_VERSION)
```

Comment:
These arguments are only for windows, so they should be appended only when `make build-windows` ?

Reply:
Updated. Use "WIN_BUILD_ARGS" for windows building.